rbac/internal/migrations/replicate_workspaces.py

```
@@ -249,11 +253,27 @@ def _replicate_deleted_batch(replicator: RelationReplicator, entries: list[_Dele
 
 
+_deleted_workspace_prefix = "Deleted workspace: "
+
+
+def _extract_deleted_workspace_name(log: AuditLog) -> str:
+    description: str = log.description
+
+    if description.startswith(_deleted_workspace_prefix):
+        return description[len(_deleted_workspace_prefix) :]
+
+    return f"[deleted workspace {str(log.resource_uuid)}]"
```

Ruff (0.16.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)